src/components/Api.js

```diff
@@ -1,185 +1,111 @@
 export default class Api {
   constructor(options) {
     // constructor body
-    this.options = options;
+    this._baseUrl = options.baseUrl;
+    this._headers = options.headers;
+  }
+
+  _checkResponse(res) {
+    if (res.ok) {
+      return res.json();
+    }
+    return Promise.reject(`Error: ${res.status}`);
+  }
+
+  _request(url, options) {
+    return fetch(url, options).then(this._checkResponse);
   }
 
   getInitialCards() {
-    return fetch("https://around.nomoreparties.co/v1/cohort-3-en/cards/", {
-      method: "GET",
-      headers: {
-        authorization: "b685d3e0-616a-4dae-bc5b-53892a4f7953",
-      },
-    }).then((res) => {
-      if (res.ok) {
-        return res.json();
+    return this._request(
+      "https://around.nomoreparties.co/v1/cohort-3-en/cards/",
```

Comment:
According to the task you need to pass `baseUrl` into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . 

You need to use `this._baseUrl` in every request not to duplicate it